docs/UserGuide.md

```diff
@@ -67,13 +67,90 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
 
 Shows a message listing all the main commands and explaining how to access the user guide.
 
-Precise command format: `help`
+#### Format: `help`
 
-Precise expected outcome on success:
-A pop-up of a new window that lists out all the main commands with their respective details and displays a link to FAPro’s user guide.
+![help format](images/helpFormat.png)
+
+#### Precise expected outcome on success:
+* Successful message: Opened help window
+* A pop-up window lists all the main commands and displays a link to FAPro's user guide
 
 ![help window](images/helpWindow.png)
 
+### Viewing risk assessment questionnaire: `questionnaire`
+
+Displays questions that are going to be used for generating client risk profile levels.
```

Comment:
I think more concise language can be used. What do you think of changing it to "displays questions that are used for ..."

Reply:
I have changed it. Thank you for the suggestion!